generators/app/index.js

```diff
@@ -93,7 +84,7 @@ module.exports = yeoman.Base.extend({
       this.props.projectClassName = toClassName(this.props.projectName);
       this.props.titlecaseProjectName = toTitleCase(this.props.projectName.replace('-', ' '));
 
-      this.props.cssClassName = toCssClassName(this.props.repoPrefix, this.props.moduleName);
+      this.props.cssClassName = _.camelCase(this.props.moduleName);
```

Comment:
Is camelCase the format we want to follow? In terra-clinical css names have been using the format `aaa-bbb`